source/agora/test/EnrollmentManager.d

```diff
@@ -0,0 +1,78 @@
+/*******************************************************************************
+
+    Contains tests for the creation of an enrollment data, enrolling as a
+    validator and propagating the information through the network
+
+    Copyright:
+        Copyright (c) 2019 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+module agora.test.EnrollmentManager;
+
+version (unittest):
+
+import agora.common.Amount;
+import agora.common.EnrollmentManager;
+import agora.common.Hash;
+import agora.consensus.data.Block;
+import agora.consensus.data.Enrollment;
+import agora.consensus.data.Transaction;
+import agora.consensus.Genesis;
+import agora.test.Base;
+
+/// test enrollment process
+unittest
+{
+    import std.algorithm;
+    import std.conv;
+    import core.time;
+    import core.thread;
```

Comment:
The `core.thread` import is not necessary. The other ones should be moved to global scope.